src/TestFramework/PhpUnit/Config/XmlConfigurationManipulator.php

```diff
@@ -117,13 +126,16 @@ public function deactivateStderrRedirection(SafeDOMXPath $xPath): void
         $this->setAttributeValue($xPath, 'stderr', 'false');
     }
 
-    public function setStopOnFailure(SafeDOMXPath $xPath): void
+    public function setStopOnFailure(string $version, SafeDOMXPath $xPath): void
```

Comment:
minor: what do you think about renaming this method to `setStopOnFailureOrDefect`?